.github/workflows/push.yml

```
@@ -314,6 +329,9 @@ jobs:
           DOCKER_BASE_IMAGE_REGISTRY: ${{ env.DOCKER_BASE_IMAGE_REGISTRY }}
         run: |
           set -x
```

actionlint (1.7.4)

property "cache_image" is not defined in object type {build_chunks: string; cache_key: string; deploy_feature: string; docker_tag: string; images: string; last_good_build_docker_tag: string; node-modules-hash: string; node_image_version: string; test_chunks: string}

(expression)